react/inputs/StyleguideInput/GenderInput.js

```diff
@@ -73,7 +74,7 @@ class GenderInput extends Component {
 
     return (
       <div
-        className={`vtex-profile-form__gender ${field.hidden ? 'dn' : ''} pb7`}
+        className={`${styles.styleguideInput} ${styles.genderInput} ${field.hidden ? 'dn' : ''} pb7`}
```

Comment:
same

Reply:
Besides, this one is a dropdown. Is it ok for it to have the class (`styles.styleguideInput`) as the input in the previous comment?